airbyte-ci/connectors/metadata_service/lib/metadata_service/models/generated/ConnectorRegistryDestinationDefinition.py

```diff
@@ -79,6 +79,36 @@ class Config:
     ql: Optional[Literal[100, 200, 300, 400, 500, 600]] = None
 
 
+class GitInfo(BaseModel):
+    class Config:
+        extra = Extra.forbid
+
+    commit_sha: Optional[str] = Field(
+        None,
+        description="The git commit sha of the last commit that modified this file. DO NOT DEFINE THIS FIELD IN YOUR SPEC. It will be overwritten by the CI.",
```

Comment:
Nit: should we lift the autogeneration notice up one/two levels and just say that the whole source_file_info probably should not be defined in a spec?